src/app/templates/app/ntia_conformance_checker.html

```diff
@@ -8,44 +8,49 @@
 {% block body1 %}
 <div id="messages" class="messages">
 </div>
-<p class ="lead"> {{ error }}</p>
+<p class ="lead">{{ error }}</p>
 <div class="panel panel-default">
 <div class="panel-heading"> <p class="lead">Tag-Value, RDF, JSON, Spreadsheet, XML SPDX Document</p> </div>
 <div class="panel-body">
-	<form id="checkform" enctype="multipart/form-data" class="form-horizontal" method="post">
-	  {% csrf_token %}
-	  <div class = "form-group">
-			&nbsp; File Type &nbsp;
-				<select name="format" id="format">
-					  <option value="0" data-value="0" selected="">-</option>
-					  <option value="XLSX">XLSX Spreadsheet</option>
-					  <option value="XLS">XLS Spreadsheet</option>
-					  <option value="RDFXML">RDF/XML</option>
-					  <option value="TAG">Tag/Value</option>
-					  <option value="JSON">JSON</option>
-					  <option value="YAML">YAML</option>
-					  <option value="XML">XML</option>
-                </select>
-		</div>
-		<div id="drop-area" class="container">
-		  <h4>Upload file using the button or by dragging and dropping onto the dashed region </h4>
-		  <input type="file" id="file" onchange="handleFiles(this.files)">
-		  <label class="button" for="file">Select file</label>
-		  <p class="file-name">No file selected</p>
-		</div>
-    </form>
+  <form id="checkform" enctype="multipart/form-data" class="form-horizontal" method="post">
+    {% csrf_token %}
+    <div class = "form-group">
+      &nbsp; File Type &nbsp;
+      <select name="format" id="format">
+        <option value="0" data-value="0" selected="">-</option>
+        <option value="XLSX">XLSX Spreadsheet</option>
```

Comment:
```suggestion
```

This option isn't supported by the Python libraries - we should remove it from the dropdown options.